merlin-server/sql/merlin/tables/plan_dataset.sql

```diff
@@ -1,6 +1,7 @@
 create table plan_dataset (
   plan_id integer not null,
   dataset_id integer not null,
+  associated_simulation_dataset_id integer default null,
```

Comment:
Minor, may be ignored: 
```suggestion
  simulation_dataset_id integer default null,
```

Reply:
While wordy, I think the original naming scheme emphasizes the optional association aspect of this new column, vs it being required.

Reply:
I'd argue that the nullability of the column is sufficient to indicate that it is optional. Additionally, the proposed name matches the precedent for other optional "associated" columns (`activity_directive.anchor_id`/`constraints.model_id`/`constraint.plan_id`).